settings/src/main/java/org/odk/collect/settings/keys/ProjectKeys.kt

```diff
@@ -73,4 +73,7 @@ object ProjectKeys {
     const val BASEMAP_SOURCE_USGS = "usgs"
     const val BASEMAP_SOURCE_STAMEN = "stamen"
     const val BASEMAP_SOURCE_CARTO = "carto"
+
+    const val GOOGLE_DRIVE_DEPRECATION_LEARN_MORE_CLICKED = "gd_lear_more_clicked"
+    const val GOOGLE_DRIVE_DEPRECATION_BANNER_DISMISSED = "gd_banner_dismissed"
```

Comment:
Storing this state as a project setting does mean that resetting all settings, then setting the project up as a Google Drive project will that the user would see the banner again even if they've already dismissed. I think this totally OK though. Just wanted to note that for @getodk/testers.